src/org/usfirst/frc4904/sovereignty/PIDContainerOrchestrator.java

```diff
@@ -0,0 +1,54 @@
+package org.usfirst.frc4904.sovereignty;
+
+
+import java.util.HashMap;
+import java.util.Map;
+import org.usfirst.frc4904.sovereignty.PIDContainer.PIDContainerModifier;
+
+public class PIDContainerOrchestrator {
+	private static PIDContainerOrchestrator singleton;
+	private final Map<String, PIDContainer> containers;
+
+	private PIDContainerOrchestrator() {
+		containers = new HashMap<String, PIDContainer>();
+	}
+
+	public static PIDContainerOrchestrator getInstance() {
+		if (PIDContainerOrchestrator.singleton == null) {
+			PIDContainerOrchestrator.singleton = new PIDContainerOrchestrator();
+		}
+		return PIDContainerOrchestrator.singleton;
+	}
+
+	public PIDContainer createContainer(String system, boolean locked) {
+		PIDContainer container = new PIDContainer(system, locked);
+		addContainer(container);
+		return container;
+	}
+
+	public void addContainer(PIDContainer container) {
+		containers.put(container.getSystem(), container);
+	}
+
+	public void addContainer(String system, PIDContainer container) {
+		containers.put(system, container);
+	}
+
+	public void getContainer(String system) {
+		containers.get(system);
+	}
+
+	public void pushValues(PIDContainerModifier modifier) {
+		containers.forEach((key, value) -> {
+			value.pushValues(modifier);
+		});
+	}
+
+	public void pullValues(PIDContainerModifier modifier) {
+		containers.forEach((key, value) -> {
```

Comment:
Since you are only using the value, consider something like `containers.valueSet().stream().filter(value->!value.isLocked()).forEach(value->value.pullValues(modifier));`